pkg/maps/lbmap/lbmap.go

```diff
@@ -117,8 +119,15 @@ func (lbmap *LBBPFMap) UpsertService(p *UpsertServiceParams) error {
 		svcVal.SetRevNat(int(p.ID))
 		svcKey.SetBackendSlot(slot)
 		if err := updateServiceEndpoint(svcKey, svcVal); err != nil {
-			return fmt.Errorf("Unable to update service entry %+v => %+v: %s",
-				svcKey, svcVal, err)
+			if errors.Is(err, unix.E2BIG) {
+				return fmt.Errorf("Unable to update service entry %+v => %+v: "+
+					"Unable to update element for LB bpf map: "+
+					"You can resize it with the flag \"--%s\". "+
+					"The resizing might break existing connections to services",
+					svcKey, svcVal, option.LBMapEntriesName)
+			}
```

Comment:
I feel like we're mixing two different things here, on one hand there's the error message and on the other hand there's a hint about how to handle that error message.

What I'd expect is that we have structured logging messages somewhere that say something like:

```
scopedLog := log
if errors.Is(err, unix.E2BIG) {
    scopedLog = scopedLog.WithField("hint", fmt.Sprintf("You can resize the loadbalancer map size with the flag \"--%s\. Resizing the map might break existing connections to services.", option.LBMapEntriesName))
}
scopedLog.WithError(err).Warn("Cannot update service")
```

Is there a reason we can't do this?